spec/lib/video_info/providers/youtube_api_spec.rb

```diff
@@ -0,0 +1,392 @@
+require 'spec_helper'
+
+describe VideoInfo::Providers::Youtube do
+  before(:all) do
+    VideoInfo.provider_api_keys = { youtube: 'AIzaSyA6PYwSr1EnLFUFy1cZDk3Ifb0rxeJaeZ0' }
+  end
+
+  describe ".usable?" do
+    subject { VideoInfo::Providers::Youtube.usable?(url) }
+
+    context "with youtube.com url" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.